web/.prettierignore

```diff
@@ -5,5 +5,8 @@
 yarn.lock
 
 world.json
+world-aggregated.json
+excluded-aggregated-exchanges.json
+Dockerfile
```

Comment:
Why ignore the Dockerfile? (or rather why didn't we do that before if it's necessary?)

Reply:
The precommit hook runs prettier locally on modified files and can't identify the correct parser for Dockerfile. I'm unsure how it passed in CI. It's a small file, hopefully we can keep it pretty :)

Reply:
I added a dev dependency to support dockerfile with prettier

Reply:
> I added a dev dependency to support dockerfile with prettier

I don't think that will work with the current CI system since we just run prettier on the whole repo with NPX.

We would have to create a top level package.json and add it there too for it to work (and change the CI to install them).

Reply:
I think the NPX prettier must be able to figure out what parser to use for Dockerfile as it's been running on it previously, this dev dep was to allow us to run the husky precommit prettier hook locally.